app/components/state_tag_component.rb

```diff
@@ -0,0 +1,16 @@
+class StateTagComponent < ApplicationComponent
+  attr_reader :claim
+
+  def initialize(claim:, classes: [], html_attributes: {})
+    super(classes:, html_attributes:)
+
+    @claim = claim
+  end
+end
+
+def state_tag
```

Comment:
Also, if this doesn't really require a template, you can name this method `call` and it will render `call` instead of the template.

Reply:
In that case, would it make more sense to bring this render logic to the template file itself?